README.md

```diff
@@ -1,26 +1,73 @@
-# Nimiq C# Client
+Nimiq C# Client
+===============
 
 > C# implementation of the Nimiq RPC client specs.
 
 ## Usage
 
-*Placeholder; Rather than starting to delve into the API docs, it'd be great to see what the client looks like in action in this lang plus a rough description what it does. The dev should be able to quickly determine whether the example fits the desired style and problem.*
+Send requests to a Nimiq node with `NimiqClient` object.
+
+```c#
+var config = new Config(
+    scheme: "http",
+    host: "127.0.0.1",
+    port: 8648,
+    user: "luna",
+    password: "moon"
+);
+
+var client = new NimiqClient(config)
+```
+
+Once the client have been set up, we can call the methods with the appropiate arguments to make requests to the Nimiq node.
+
+When no `config` object is passed in the initialization it will use the default values in the Nimiq node.
+
+```c#
+var client = NimiqClient();
+
+// make rpc call to get the block number
+var blockNumber = await client.blockNumber();
+
+Console.WriteLine(blockNumber) // displays the block number, for example 748883
+```
 
 ## API
 
-*Placeholder; All the implemented methods with parameters, types, default values, return types*
+The complete [API documentation](docs) is available in the `/docs` folder.
+
+Check out the [Nimiq RPC specs](https://github.com/nimiq/core-js/wiki/JSON-RPC-API) for behind the scene RPC calls.
 
 ## Installation
 
-*Placeholder; how to add this client to a project. Each language comes with a package manager. To be useful for developers, add a package description file that works with the most commonly used package manager for this language. For example have a `package.json` file for JavaScript or `pom.xml` for Java.*
+To install NimiClient add the Nuget package as a dependency to your project from Visual Studio IDE.
 
 ## Contributions
 
-This implementation was originally contributed by [*Placeholder; Your name with link to GitHub*](https://github.com/nimiq/).
+This implementation was originally contributed by [rraallvv](https://github.com/rraallvv/).
 
 Please send your contributions as pull requests.
+
 Refer to the [issue tracker](https://github.com/nimiq-community/csharp-client/issues) for ideas.
 
+### Develop
+
+After cloning the repository, open the solucion file in the repository root folder.
```

Comment:
can not fine a "solucion" file.

Reply:
[Added](https://github.com/rraallvv/csharp-client/commit/4f28379fc27cfe940665f7368e831f6a28de659e) the name of the solution file to the README.md file